Product/SoapUI_Test/pom.xml

```diff
@@ -24,8 +23,10 @@
         <as.username>admin</as.username>
         <as.password>adminadmin</as.password>
         <jmx.host>localhost</jmx.host>
-        <jmx.port>8686</jmx.port>
+        <jmx.port>9990</jmx.port>
```

Comment:
I'm ok with this change.  But just asking, does this mean we no longer are going to be using the 8686 port?